apis/beacon/pool/validator_registration.yaml

```diff
@@ -0,0 +1,21 @@
+get:
+  operationId: getValidatorRegistrationV1
+  summary: Get validator registrations
+  description: Retrieves validator registrations for builder api.
+  tags:
+    - Beacon
+    - ValidatorRequiredApi
+  responses:
+    "200":
+      description: Success
+      content:
+        application/json:
+          schema:
+            title: ValidatorRegistrationV1
+            type: array
```

Comment:
I was thinking about making this a specific endpoint with /$validator at the end however all the clients submit the registrations in one big bundle so I thought it made sense to just return an array of registrations for all validators associated with that beacon